osquery/mdm.go

```diff
@@ -0,0 +1,113 @@
+package osquery
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+
+	"github.com/go-kit/kit/log"
+	"github.com/groob/plist"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+func MDMInfo(logger log.Logger) *table.Plugin {
+	columns := []table.ColumnDefinition{
+		table.TextColumn("enrolled"),
+		table.TextColumn("server_url"),
+		table.TextColumn("checkin_url"),
+		table.IntegerColumn("access_rights"),
+		table.TextColumn("install_date"),
+		table.TextColumn("payload_identifier"),
+		table.TextColumn("topic"),
+		table.TextColumn("sign_message"),
+		table.TextColumn("identity_certificate_uuid"),
+		table.TextColumn("has_scep_payload"),
+	}
+	return table.NewPlugin("kolide_mdm_info", columns, generateMDMInfo)
+}
+
+func generateMDMInfo(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	profiles, err := getMDMProfile()
+	if err != nil {
+		return nil, err
+	}
+
+	var enrollProfileItems []profileItem
+	var results []map[string]string
+	var mdmResults map[string]string
+	for _, payload := range profiles.ComputerLevel {
+		for _, item := range payload.ProfileItems {
+			if item.PayloadContent == nil {
+				continue
+			}
+			if item.PayloadType == "com.apple.mdm" {
+				enrollProfile := item.PayloadContent
+				enrollProfileItems = payload.ProfileItems
+				mdmResults = map[string]string{
+					"enrolled":                  "true",
+					"server_url":                enrollProfile.ServerURL,
+					"checkin_url":               enrollProfile.CheckInURL,
+					"access_rights":             fmt.Sprintf("%d", enrollProfile.AccessRights),
+					"install_date":              payload.ProfileInstallDate,
+					"payload_identifier":        payload.ProfileIdentifier,
+					"sign_message":              fmt.Sprintf("%v", enrollProfile.SignMessage),
```

Comment:
If we are preferring `strconv` to `fmt`, I think we should use `strconv.FormatBool` here.